tests/resources/bootstrap-jpm.js

```diff
@@ -0,0 +1,385 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a test framework used in the tests to make sure it's correctly detected as a framework, right? If so, could it be much shorter, maybe even just containing a line saying that it's a test framework?

In any case, it needs a comment at the beginning of this file to give the context, and explain it's not to be used/updated/...